app/models/provider.rb

```diff
@@ -59,29 +47,11 @@ def firm_permissions
     firm.nil? ? [] : firm.permissions
   end
 
-  # TODO: AP-6181: will need to change this or remove/replace entirely
-  def ccms_apply_role?
-    return true if Rails.configuration.x.omniauth_entraid.mock_auth == "true"
-    return true if auth_provider.eql?("entra_id") && auth_subject_uid.present?
-
-    return false if roles.nil?
-
-    roles.split(",").include?("CCMS_Apply")
-  end
-
   def invalid_login?
     invalid_login_details.present?
   end
 
-  def newly_created_by_devise?
-    firm_id.nil?
-  end
-
   def provider_details_api_error?
```

Comment:
I think this can go too... it was only ever set in the provider_after_login_service
The def invalid_login? is also not needed, I think?
The underlying DB field could possibly go too?